js/app.js

```diff
@@ -0,0 +1,182 @@
+// 1
+function setAlarm(employed, vacation){
+    return employed && vacation === false;
```

Comment:
goog, but use the logical Not `!` operator instead of equality operator along with `false`